spec/classes/init_spec.rb

```diff
@@ -12,135 +12,148 @@
         it { is_expected.to contain_class('gdm::service') }
 
         context 'EL7 with GDM = 3.0.0' do
-          if os_facts[:operatingsystemmajrelease].to_s >= '7'
-            let(:facts){ os_facts.merge({:runlevel => '5', :gdm_version => '3.14.2'}) }
+          let(:facts){ os_facts.merge({:runlevel => '5', :gdm_version => '3.14.2'}) }
```

Comment:
Should the context above be `With GDM = 3.0.0` instead?

Reply:
Changed the comment